packages/wallet-sdk/src/sign/walletlink/WalletLinkSigner.ts

```diff
@@ -303,8 +299,10 @@ export class WalletLinkSigner implements Signer {
     const params = request.params || [];
 
     switch (method) {
-      case 'eth_requestAccounts':
-        return this._eth_requestAccounts();
+      case 'eth_requestAccounts': {
+        await this.handshake();
+        return { jsonrpc: '2.0', id: 0, result: this._addresses };
```

Comment:
just following the old code adding this nonsense `jsonrpc: '2.0', id: 0,` 
![image](https://github.com/user-attachments/assets/3868825e-d0c1-48cc-bbae-33f952be569b)
we don't use them anywhere, so it's probably okay to deprecate. will revisit this when we refactor WalletLink implementation